src/test/regress/expected/multi_explain.out

```diff
@@ -6,7 +6,7 @@ ALTER SEQUENCE pg_catalog.pg_dist_shardid_seq RESTART 570000;
 SELECT substring(version(), '\d+(?:\.\d+)?') AS major_version;
  major_version 
 ---------------
- 10
+ 9.6
```

Comment:
we should not be changing this.
multi_explain.out is for PG 10
multi_explain_0.out is for PG 9.6